tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -573,6 +573,7 @@ TEST_F(FileTest, IntrinsicsFloatSign) {
   runFileTest("intrinsics.floatsign.hlsl");
 }
 TEST_F(FileTest, IntrinsicsIntSign) { runFileTest("intrinsics.intsign.hlsl"); }
+TEST_F(FileTest, IntrinsicsRcp) { runFileTest("intrinsics.rcp.hlsl"); }
```

Comment:
It would be nice to add tests for double constants in https://github.com/Microsoft/DirectXShaderCompiler/blob/master/tools/clang/test/CodeGenSPIRV/constant.scalar.hlsl.